tests/acceptance/bootstrap/WebDav.php

```diff
@@ -1573,32 +1573,34 @@ public function checkElementList(
 				//       /some-folder%20with%20spaces/sub-folder
 				// So we need both $elementToRequest and $expectedElement
 				$expectedElement = $this->encodePath($elementToRequest);
+
 				$responseXmlObject = $this->listFolderAndReturnResponseXml(
 					$user,
 					$elementToRequest,
 					"1"
 				);
 
-				// TODO: make it work for folder entries
-				// Doesn't work for folder entries
-				// as the folder entry has trailing '/' in d:href
 				$webdavPath = "/" . $this->getFullDavFilesPath($user) . $expectedElement;
+				// return xmlobject that matches the x-path pattern.
```

Comment:
It should clarify that we will only check the first element because the requested resource will always be the first one
```suggestion
				// return xmlobject that matches the x-path pattern.
```